Source: maxice8
Upstream: https://github.com/libratbag/libratbag/pull/507
Reason: fixes errors of PATH_MAX being undeclared due to limits.h not being included

--- src/libratbag-data.c
+++ src/libratbag-data.c
@@ -28,6 +28,7 @@
 #include <assert.h>
 #include <stdlib.h>
 #include <glib.h>
+#include <limits.h>
 
 #include "libratbag.h"
 #include "libratbag-private.h"
--- tools/lur-command.c
+++ tools/lur-command.c
@@ -31,6 +31,7 @@
 #include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <limits.h>
 
 #include <liblur.h>
 #include <libratbag-util.h>
